plugins/woocommerce/client/blocks/assets/js/blocks/cart/test/block.js

```
@@ -147,63 +161,66 @@ describe( 'Testing cart', () => {
 
-		await waitFor( () => expect( fetchMock ).toHaveBeenCalled() );
-		expect( screen.getAllByRole( 'cell' )[ 1 ] ).toHaveTextContent( '16€' );
+		await waitFor( () =>
+			expect( screen.getAllByRole( 'cell' )[ 1 ] ).toHaveTextContent(
+				'16€'
+			)
+		);
 	} );
 
-	it( 'updates quantity when changed in server', async () => {
+	it.only( 'updates quantity when changed in server', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)